packages/frontend/src/components/cart/Cart.tsx

```diff
@@ -59,27 +61,27 @@ export default function Cart({
   }
 
   useEffect(() => {
-    if (!currentOrder || !stateManager) return;
-    logger.debug(`Showing cart items for order ID: ${currentOrder.ID}`);
-    stateManager.get(["Orders", currentOrder.ID])
+    if (!activeOrder || !stateManager) return;
+    logger.debug(`Showing cart items for order ID: ${activeOrder.ID}`);
+    stateManager.get(["Orders", activeOrder.ID])
       .then(async (res: CodecValue | undefined) => {
         if (!res) {
-          throw new Error("No order found");
+          return logError("No order found");
```

Comment:
this is a bit unorthodox. unless logError should actually return something to the next promise, do:

```suggestion
          logError("No order found");
          return;
```